src/test/java/uk/gov/hmcts/cmc/claimstore/rules/MoreTimeRequestRuleTest.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.hmcts.cmc.claimstore.rules;
+
+import org.junit.Test;
+import uk.gov.hmcts.cmc.claimstore.exceptions.MoreTimeAlreadyRequestedException;
+import uk.gov.hmcts.cmc.claimstore.exceptions.MoreTimeRequestedAfterDeadlineException;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleClaim;
+
+import java.time.LocalDate;
+
+public class MoreTimeRequestRuleTest {
+
+    private final MoreTimeRequestRule moreTimeRequestRule = new MoreTimeRequestRule();
+
+    @Test
+    public void shouldNotThrowExceptionWhenMoreTimeRequested() {
```

Comment:
The test case description is not right. When more time requested it should thrown an exception. On the other side test case tests scenario where no exception is thrown due to the fact this is a first request.

Please correct.

Reply:
done